tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -1007,8 +1007,20 @@ def i(...):
       return ParallelMapDataset(self, map_func, num_parallel_calls)
 
   def flat_map(self, map_func):
-    """Maps `map_func` across this dataset and flattens the result. Will produce identical results to 'tf.data.Dataset.interleave'
+    """Maps `map_func` across this dataset and flattens the result. 
+    
+    Will produce similar results to `tf.data.Dataset.interleave(cycle_length=1)`. 
```

Comment:
It's probably easier for new users to mention `interleave` as a generalization of `flat_map`, instead of `flat_map` as a simplification of `interleave.`